File: src/Example3Java/EX18.java
package Example3Java;

import java.util.Scanner;

public class EX18 {
    public static void main(String[] args) {
        System.out.println("Enter number : ");
        int x = new Scanner(System.in).nextInt();
        int[] num = {1, 2, 3, 4, 5, 6, 33, 6, 8, 33};
        int sum = 0;
        for (int i = 0; i < num.length; i++) {
            if (num[i] == x) {
                sum++;
            }
        }
        System.out.println("sum Array x == num : " + sum);
    }
}

File: src/Example3Kotlin/EX18.kt
import java.util.*

object EX18 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter number : ")
        val x = Scanner(System.`in`).nextInt()
        val num = listOf(1, 2, 3, 4, 5, 6, 33, 6, 8, 33)
        var sum = 0
        for (i in num.indices) if (num[i] == x) sum++
        println("sum Array x == num : $sum")
    }
}
